hcloud/load_balancer.go

```diff
@@ -119,6 +119,25 @@ type LoadBalancerTargetType string
 const (
 	// LoadBalancerTargetTypeServer is a target type which points to a specific server.
 	LoadBalancerTargetTypeServer LoadBalancerTargetType = "server"
+
+	// LoadBalancerTargetTypeLabelSelector is a target type which points to a label selector.
+	LoadBalancerTargetTypeLabelSelector LoadBalancerTargetType = "label_selector"
+
+	// LoadBalancerTargetTypeIP is a target type which points to an IP.
+	//
+	// Using IP targets makes it possible to use the Hetzner Cloud Load
+	// Balancer together with Hetzner Online dedicated Root Servers.
+	//
+	// The following restrictions apply for the IP:
+	//
+	//   * The IP must be owned by the project owner.
+	//
+	//   * The IP must not point to an Hetzner Cloud resource.
+	//
+	// In sum the above rules limit the IP target to Hetzner Online dedicated
+	// Root Servers owned by the same account as the Hetzner Cloud Project to
+	// which the Load Balancer belongs.
```

Comment:
I think this is too detailed and does not belong into the library’s documentation, but rather in the public API docs at docs.hetzner.cloud.